README.md

```diff
@@ -1,156 +1,14 @@
-v0.14.0-f6da9c76877551ef32503b17189bb178501f59a7 v0.14.0 UNKNOWN
-```
-
-Here we can see a single release in the bucket, version `v0.14.0`.
-The git commit ref is also included as part of the name, so you can be sure
-that the correct revision of cert-manager has in fact been built.
-
-Once you have found the release you wish to stage in this list, make a note of
-the release's `name` and proceed to step 3!
-
-#### Step 3 - publishing a staged release
-
-Once a release has been staged into the release bucket and we've verified it
-has been built from the correct revision of cert-manager, we are now ready to
-trigger the publishing stage of the release.
-
-In this step, the staged release is fetched from Google Cloud Storage,
-validated, and then pushed out to public facing locations.
-
-```
-
-$ cmrel publish \
-    --release-name v0.14.0-f6da9c76877551ef32503b17189bb178501f59a7 \
-    --nomock
-```
-
-If you do not specify the `--nomock` flag, `cmrel` will *not* push any
-artifacts and will only fetch and validate the release before exiting.
-
-The final stage of this step is to create a GitHub release in the cert-manager
-repository, as well as uploading 'static manifests' to the release.
-To allow you to update the release with proper release notes
-*before publishing*, `cmrel` will mark the created release as a **DRAFT**.
-You must then edit the release to include the appropriate release notes, and
-then hit 'Publish'!
-
-If you are intending to publish to your own, private release buckets (i.e. to
-test this whole workflow, or for creating internal releases) you should be sure
-to set the following flags when calling `cmrel publish`:
-
-```
-    --published-image-repo='quay.io/mycompany' # prefix for images, e.g. 'quay.io/mycompany'
-    --published-helm-chart-bucket='mycompany-helm-charts' # name of the GCS bucket where the built Helm chart should be stored
-    --published-github-org='mycompany' # name of the GitHub org containing the repo that will be tagged at the end
-```
-
-### Development
-
-#### Creating development builds
-
-By default the artifacts created during a release process are pushed to `cert-manager-release` bucket at `/stage/gcb/release` path.
-It is also possible to create a 'development' build by skipping the `--release-version` flag on `cmrel stage` command. This will result in the build artifacts being pushed to `cert-manager-release` bucket at `/stage/gcb/devel` path.
-
-If you have made some local changes to this tool and want to create a 'devel' build to test them, be mindful that the Google Cloud Build triggered by running `cmrel stage` clones this repository from GitHub and runs its own `cmrel` commands. You can modify the [Cloud Build config](https://github.com/cert-manager/release/blob/master/gcb/stage/cloudbuild.yaml) to configure a different GitHub repository/branch.
+Documentation for the release process is on the [cert-manager website](https://cert-manager.io/docs/contributing/release-process/).
```

Comment:
Suggestion: this is opinionated, but I would rather have these docs here (and perhaps less info on the website and/or the website doc could have a link to this Readme). The reasons would  be:
- this Readme contains `cmrel` development/debugging related info that is not present in the website and probably shouldn't be, so we don't make a person who wants to quickly release something read through debugging info
- it's easier to link code changes to Readme changes when they're in the same repo
- If we use `cmrel` to publish also other projects (I seem to remember this mentioned) it'd make sense to have in-detail docs in a common place
- Once we have the release process more automated, it'd arguably make sense to remove information related to the automated steps from website and keep it just here for `cmrel` development purposes

Reply:
I think that's a super reasonable position to take, yeah. Rather than address that now, what I'll do is remove most of my changes to the README and keep this PR for the trivial code changes.

Reply:
Makes sense to me. I agree with the general idea that the docs are duplicated here and on the website and shouldn't be so 👍🏼